src/android/nl/xservices/plugins/SocialSharing.java

```diff
@@ -9,6 +9,7 @@
 import android.net.Uri;
 import android.os.Build;
 import android.os.Environment;
+import android.support.annotation.NonNull;
```

Comment:
Hey @mbektchiev can we live without this annotation? I'd rather not drag in a support library for this..